src/renderers/testing/ReactTestRenderer.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactTestRenderer
+ * @flow
```

Comment:
@gaearon should I add flow as a separate PR? I know you mentioned keeping style changes separate from feature changes, but I wasn't sure where flow landed on that spectrum.